### PR TITLE
fix(#7039): tmp fix AbstractConfigChangeListener parseChangeData not right content bug.

### PR DESCRIPTION
**Pull request by CherishCai**
fix(#7039): tmp fix AbstractConfigChangeListener parseChangeData not right content bug.

image

### CODE REVIEW

File: client/src/main/java/com/alibaba/nacos/client/config/impl/CacheData.java [hunk 1]
```diff
@@ -226,7 +235,7 @@ public void run() {
                             name, dataId, group, md5, listener, ex.getErrCode(), ex.getErrMsg());
                 } catch (Throwable t) {
                     LOGGER.error("[{}] [notify-error] dataId={}, group={}, md5={}, listener={} tx={}", name, dataId,
-                            group, md5, listener, t.getCause());
+                            group, md5, listener, t.getCause(), t);
```
Comment (CherishCai) [inline client/src/main/java/com/alibaba/nacos/client/config/impl/CacheData.java:238]: print unknown error detail

**Comment by KomachiSion** [comment]
CLA no signed for a long time.

**Comment by CherishCai** [comment]
I'm sorry it's late to sign the CLA.

**Comment by CherishCai** [comment]
@KomachiSion Will you reopen it?

**Pull request closed without merging**